packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -408,6 +429,12 @@ export class DocumentsSchemaController {
 	) {
 		checkRuntimeCompatibility(documentMetadataSchema);
 
+		// For simplicity, let's only support new schema features for explicit schema control mode
+		assert(
+			features.disallowedVersions.length === 0 || features.explicitSchemaControl,
```

Comment:
I do not understand how `disallowedVersions` will be used. Can you maybe help me understand with a couple of scenarios? The tests kind of do that but I could not follow what the tests are doing.

Reply:
I've added some comments (on a type) and in other places - can you please take a look if it answers your question?